layouts/section/api.html

```diff
@@ -165,6 +166,157 @@
+              <h5>Example Response</h5>
+              {{ partial "snippet-result-code-block" (dict "context" . "file" "api-billing-usage-hosts.sh" ) }}
+            </div>
+        </div>
+        </div>
+
+        <div class="int-anchor" id="billing-usage-timeseries"></div><a href="#billing-usage-timeseries"><h4>Get Hourly Usage For Custom Metrics.</h4></a>
+        <div class="row">
+          <div class="col-xs-12 col-md-6 api-left">
+            <p>
+              Get Hourly Usage For Custom Metrics. This endpoint is not available in the Python and Ruby libraries.
+            </p>
+            <div class="c">The most recent hourly usage data is 3 days old</div>
+            <h5>Arguments</h5>
+            <ul class="arguments">
+            {{ partial "argument.html" (dict "context" $dot "name" "start_hr" "description" "datetime in ISO-8601 format, UTC, precise to hour: [YYYY-MM-DDThh] for usage beginning at this hour (UTC)" "lang" "console")}}
+            {{ partial "argument.html" (dict "context" $dot "name" "end_hr" "description" "datetime in ISO-8601 format, UTC, precise to hour: [YYYY-MM-DDThh] for usage ending BEFORE this hour (UTC)" "lang" "console" "default" "1d+start_hr")}}
+            </ul>
+          </div>
+          <div class="col-xs-12 col-md-6 api-right">
+            <h5>Signature</h5>
+            <code>GET https://app.datadoghq.com/api/v1/usage/timeseries</code>
+            <h5>Example Request</h5>
+            {{ partial "snippet-code-block" (dict "context" . "file" "api-billing-usage-timeseries.sh" )}}
+            <h5>Example Response</h5>
+            {{ partial "snippet-result-code-block" (dict "context" . "file" "api-billing-usage-timeseries.sh" ) }}
+          </div>
+        </div>
+        <div class="int-anchor" id="billing-usage-top-avg-metrics"></div><a href="#billing-usage-top-avg-metrics"><h4>Get Top 500 Custom Metrics By Hourly Average</h4></a>
+        <div class="row">
+          <div class="col-xs-12 col-md-6 api-left">
+            <p>
+              Get Top Custom Metrics By Hourly Average. This endpoint is not available in the Python and Ruby libraries.
+            </p>
+            <div class="c">The most recent hourly usage data is 3 days old</div>
+            <h5>Arguments</h5>
+            <ul class="arguments">
+            {{ partial "argument.html" (dict "context" $dot "name" "start_hr" "description" "datetime in ISO-8601 format, UTC, precise to hour: [YYYY-MM-DDThh] for usage beginning at this hour (UTC)" "lang" "console")}}
+            {{ partial "argument.html" (dict "context" $dot "name" "end_hr" "description" "datetime in ISO-8601 format, UTC, precise to hour: [YYYY-MM-DDThh] for usage ending BEFORE this hour (UTC)" "lang" "console" "default" "1d+start_hr")}}
+            {{ partial "argument.html" (dict "context" $dot "name" "names" "description" "comma-separated list of metric names" "lang" "console" "default" "None")}}
```

Comment:
Data behind this endpoint is different as it is only stored by month.  I think `month` is the only suitable parameter for this endpoint?